testsuite/src/main/java/io/netty5/testsuite/transport/socket/SocketStartTlsTest.java

```diff
@@ -130,11 +131,13 @@ public static void testStartTlsNotAutoRead(ServerBootstrap sb, Bootstrap cb,
 
     private static void testStartTls(ServerBootstrap sb, Bootstrap cb,
                                      SslContext serverCtx, SslContext clientCtx, boolean autoRead) throws Throwable {
+        sb.childOption(ChannelOption.RCVBUF_ALLOCATOR_USE_BUFFER, true);
+        cb.option(ChannelOption.RCVBUF_ALLOCATOR_USE_BUFFER, true);
```

Comment:
Can we reuse `io.netty5.testsuite.transport.AbstractComboTestsuiteTest#enableNewBufferAPI`?